apps/portlet-documents/src/main/webapp/vue-app/files-search/components/FileSearchCard.vue

```diff
@@ -54,10 +54,6 @@ export default {
     },
   },
   mounted() {
-    if (this.result && this.excerptTitle) {
-      // eslint-disable-next-line vue/no-mutating-props
-      this.result.title = this.excerptTitle;
-    }
```

Comment:
In fact, by removing this, we lost the window.decodeURIComponent called on this.excerpts['title'][0]
So I think we should keep the if like this : 
if (this.result && this.excerptTitle) {
  // eslint-disable-next-line vue/no-mutating-props
  this.result.excerptTitle = this.excerptTitle;
}

and in ExoDocument.vue : 
v-sanitized-html="document.excerptTitle ? document.excerptTitle : document.title"